aiohasupervisor/client.py

```diff
@@ -94,42 +117,28 @@ async def _request(
             self._close_session = True
 
         try:
-            async with self.session.request(
+            response = await self.session.request(
```

Comment:
I'm not sure it matters, but we could separate the cases where we can use the context manager from the cases we don't:

```py
if response_type == ResponseType.RAW:
    response = await self.session.request(
        ...
    )
    ...
else:
    async with self.session.request(
        ...
    ) as response:
        ...
```